support-services/.env

```diff
@@ -1 +1 @@
-GEONETWORK_VERSION=4.2.2
+GEONETWORK_VERSION=4.2.8
```

Comment:
This is important because we are supposed to maintain compatibility with the 4.2.2 version. If there is no way to have the system work with 4.2.2 then we have the option to disable feedbacks but it has to be justified.